bumble/profiles/vocs.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# -----------------------------------------------------------------------------
+# Imports
+# -----------------------------------------------------------------------------
+import enum
+import struct
+
+from typing import Optional
+
+from bumble import att
+from bumble import device
+from bumble import gatt
+from bumble import gatt_client
+from bumble.profiles.bap import AudioLocation
+
+
+# -----------------------------------------------------------------------------
+# Constants
+# -----------------------------------------------------------------------------
+
+MIN_VOLUME_OFFSET = -255
+MAX_VOLUME_OFFSET = 255
+CHANGE_COUNTER_MAX_VALUE = 0xFF
+
+SET_VOLUME_OFFSET_OPCODE = 0x01
+
+
+class ErrorCode(enum.IntEnum):
```

Comment:
It would be better to use `utils.OpenIntEnum` here so that the code doesn't raise an exception when encountering new/unknown error codes.